language/documentation/examples/experimental/rounding-error/sources/Uniswap.move

```diff
@@ -0,0 +1,130 @@
+module NamedAddr::Uniswap {
```

Comment:
Explain example and where it comes from (error in Uniswap 1 -- URL to explanation?)